packages/components/src/components/navigation-item/navigation-item.ts

```diff
@@ -130,21 +146,22 @@ export default class SdNavigationItem extends SolidElement {
       description: this.hasSlotController.test('description'),
       children: this.hasSlotController.test('children')
     };
-    const tag = isLink ? literal`a` : isAccordion ? literal`summary` : literal`button`;
+
     const horizontalPaddingBottom = this.vertical ? 'pb-3' : 'pb-2';
 
     /* eslint-disable lit/no-invalid-html */
     /* eslint-disable lit/binding-positions */
     const root = html`
-      <${tag}
+      <${this.tag}
         part="base"
         class=${cx(
-          'px-4 hover:bg-neutral-200 group transition-all min-h-[48px] cursor-pointer relative focus-visible:focus-outline',
+          'cursor-pointer relative focus-visible:focus-outline',
           { base: 'text-base', lg: 'text-lg', sm: 'text-[14px]' }[this.size],
           this.disabled ? 'text-neutral-500 pointer-events-none' : 'text-primary',
           isAccordion ? 'flex flex-col' : 'inline-block w-full',
           this.divider && this.vertical && 'mt-0.25',
-          !this.vertical && 'inline-flex items-center'
+          !this.vertical && 'inline-flex items-center',
+          !this.separated && 'hover:bg-neutral-200 group transition-all min-h-[48px] px-4'
```

Comment:
Styles on hover and focus still feel off. Could you have a look, @coraliefeil or make an example how it should look like?


<video src="https://github.com/user-attachments/assets/3baf54e5-cbc6-4126-a8d9-706e80d8f7ff"></video>

Reply:
> Styles on hover and focus still feel off. Could you have a look, @coraliefeil or make an example how it should look like?
> 
>  CleanShot.2024-08-09.at.09.25.31-converted.mp4

@auroraVasconcelos we show the[ focus state here](https://www.figma.com/design/lDAQh9OhtODQjx0mTzdp8A/Navigation-Item?node-id=2147-1241&t=jenj0c5ovKI99iYl-11) (just ignore the direction of the chevron)
Pls adjust the implementation so that it looks more like intended. Does not need to be pixel-perfect but a bit more similar would be good. THX

When inspecting the layouts you can see that a 16px gap between the 2 interactive areas is set. Pls implement it accordingly. THX